src/99-appendices/10-file-collections.md

```diff
@@ -0,0 +1,50 @@
+
+### Fieldmap data
+
+| Suffix | Meta-data relevant entity     | Application                        | Description                                                                                                                                                                                                                                                                                                                                                                       |
+|--------|-------------------------------|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TB1DAM | flip                          | Double-angle B1+ mapping           | The double-angle B1<sup>+</sup> method ([Insko and Bolinger 1993](https://www.sciencedirect.com/science/article/abs/pii/S1064185883711332)) is based on the calculation of the actual angles from signal ratios, collected by two acquisitions at different nominal excitation flip angles. Common sequence types for this application include spin echo and echo planar imaging. |
+| TB1EPI | flip, echo                    | B1<sup>+</sup> mapping with 3D EPI | This B1<sup>+</sup> mapping method ([Jiru and Klose 2006](https://dx.doi.org/10.1002/mrm.21083)) is based on two EPI readouts to acquire spin echo (SE) and stimulated echo (STE) images at multiple flip angles in one sequence, used in the calculation of deviations from the nominal flip angle.                                                                              |
+| TB1AFI | flip, inv                     | Actual Flip Angle Imaging (AFI)    | This method ([Yarnykh 2007](https://dx.doi.org/10.1002/mrm.21120)) calculates a B1<sup>+</sup> map from two images acquired at interleaved (two) TRs with identical RF pulses using a steady-state sequence                                                                                                                                                                       |
+| TB1TFL | Please see the qMRI appendix* | Siemens `tfl_b1_map`               | B1<sup>+</sup> data acquired using `tfl_b1_map` product sequence by Siemens based on the method by [Chung et al. (2010)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.22423). The sequence generates one ~anatomical image and one scaled flip angle map.                                                                                                                 |
```

Comment:
Do we intend to get the qMRI appendix in before the next release? Could we put in another placeholder for now, or only add these entries when the qMRI appendix is added?

Reply:
@effigies merging this PR and the new entities mentioned here adds the major components of BEP001 to the spec. I think shipping the qMRI appendix in the next release adds to the clarity of these new concepts. 

I will change links to a placeholder text to prevent link-checker fails.

Reply:
> What alternative placeholder would you suggest?

I suggest that after this PR is merged you ( @agahkarakuzu ) make a new PR introducing the QMRI appendix to the spec, which you can then link to. Would that be possible relatively quickly, or do you have to prepare it first?

I remember that you said in a meeting that all of the material is basically "ready"